_docs_v5/2_configuration/interfaces.md

```diff
@@ -39,19 +41,19 @@ COSMOS provides the following interfaces for use: TCPIP Client, TCPIP Server, UD
 
 The TCPIP client interface connects to a TCPIP socket to send commands and receive telemetry. This interface is used for targets which open a socket and wait for a connection. This is the most common type of interface.
 
-| Parameter          | Description                                                                        | Required |
-| ------------------ | ---------------------------------------------------------------------------------- | -------- |
-| Host               | Machine name to connect to                                                         | Yes      |
-| Write Port         | Port to write commands to (can be the same as read port)                           | Yes      |
-| Read Port          | Port to read telemetry from (can be the same as write port)                        | Yes      |
-| Write Timeout      | Number of seconds to wait before aborting the write. Pass 'nil' to block on write. | Yes      |
-| Read Timeout       | Number of seconds to wait before aborting the read. Pass 'nil' to block on read.   | Yes      |
-| Protocol Type      | See Protocols.                                                                     | No       |
-| Protocol Arguments | See Protocols for the arguments each stream protocol takes.                        | No       |
+| Parameter          | Description                                                                                             | Required |
+| ------------------ | ------------------------------------------------------------------------------------------------------- | -------- |
+| Host               | Machine name to connect to                                                                              | Yes      |
+| Write Port         | Port to write commands to (can be the same as read port). Pass nil to make the interface read only.     | Yes      |
+| Read Port          | Port to read telemetry from (can be the same as write port). Pass nil to make the interface write only. | Yes      |
```

Comment:
This is how you make a connection read or write only right?

Reply:
yes